app/models/user.rb

```diff
@@ -2,4 +2,8 @@ class User < ApplicationRecord
   has_many :posts, foreign_key: :author_id
   has_many :comments
   has_many :likes
```

Comment:
- [ ] Kindly I recommend you to specifying that the `foreign_key` on the  **comments**, and **likes** tables that establishes the connection with the associated model is `author_id`. This means that Rails will look for an `author_id` column in these tables to set up the association. otherwise  Rails will use the convention to look for a foreign key of modelname_id (where modelname is the name of the associated model in lowercase). **So, for comments, it will look for `comment_id`, and for likes, it will look for `like_id`.**

```ruby
has_many :posts, foreign_key: 'author_id'
has_many :comments, foreign_key: 'author_id'
has_many :likes, foreign_key: 'author_id'
```